operator/v1/types_network.go

```diff
@@ -110,7 +128,10 @@ type AdditionalNetworkDefinition struct {
 
 	// rawCNIConfig is the raw CNI configuration json to create in the
 	// NetworkAttachmentDefinition CRD
-	RawCNIConfig string `json:"rawCNIConfig"`
+	RawCNIConfig string `json:"rawCNIConfig,omitempty"`
+
+	// MacVlanConfig configures the maclvan interface
+	MacVlanConfig *MacVlanConfig `json:"macVlanConfig,omitempty"`
```

Comment:
Is this optional (linked PR suggests no)? If not this should not be `omitempty`, and I would use the value instead of the pointer.

Reply:
changed.